developer-docs-site/docs/nodes/validator-node/operator/connect-to-aptos-network.md

```diff
@@ -268,6 +268,125 @@ After your validator node joined the validator set, you can verify the correctne
     
     You should expect the active value for your `StakePool` to keep increasing. It is updated at every epoch.
 
+## Seeing your stake pool information
+
+To see the details of your stake pool, run the below CLI command with the `get-stake-pool` option by providing the `--owner-address` and `--url` fields. 
+
+
+
+:::tip Use CLI 0.3.8 or higher
+Make sure you use the CLI version 0.3.8 or higher. See [Installing Aptos CLI](/cli-tools/aptos-cli-tool/install-aptos-cli.md).
+- Type `aptos --help` to see the CLI version.
+- Type `aptos node get-stake-pool --help` for more on the command option for the below example.
+:::
+
+The below command is an example for Premainnet and an example owner address `0x6064d2f4c38b65e9b78fbdf8a80f084159341d47b5e0c192492923326d1bed0a`. For other networks, use the appropriate REST URL for the `--url` field. See [Aptos Blockchain Deployments](/nodes/aptos-deployments) for `--url` field values. 
+
+```bash
+aptos node get-stake-pool \
+  --owner-address 0x6064d2f4c38b65e9b78fbdf8a80f084159341d47b5e0c192492923326d1bed0a \
+  --url https://premainnet.aptosdev.com
+```
+
+Example output:
+
+```json
+ Finished dev [unoptimized + debuginfo] target(s) in 0.84s
+     Running `target/debug/aptos node get-stake-pool --owner-address 0x6064d2f4c38b65e9b78fbdf8a80f084159341d47b5e0c192492923326d1bed0a`
+{
+  "Result": [
+    {
+      "state": "Active",
```

Comment:
Stake pool state:
1. Active: Already in validator set and proposing.
2. Pending_active: Will be added in next epoch. I've seen cases where operators complain that they got an error trying to join the validator set again.